go/iot/protocol/opencensus/openCensusBase.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opencensus
```

Comment:
Same, consider renaming to `opencensus_base.go`

Reply:
maybe opencensusbase is better?  I seldom see the file name contains "_" in this directory.

Reply:
https://github.com/census-instrumentation/opencensus-go/tree/master/stats:
`measure_float64.go`  `measure_int64.go`

Using underscore is a convention for go files.